achievements/helpingHand.achievement.js

```diff
@@ -0,0 +1,38 @@
+var _ = require('lodash');
+
+var helpingHand = {
+  name: 'Helping Hand',
+  check: function(pullRequest, shall) {
+    var committedReviewers = reviewersWhoCommittedToPullRequest(pullRequest);
+
+    if (!_.isEmpty(committedReviewers)) {
+
+      var achievement = {
+        avatar: 'images/achievements/helpingHand.achievement.jpg',
+        name: 'Helping Hand',
+        short: 'Let me help you finishing up here...',
+        description: 'You\'ve committed to a Pull Request you are reviewing',
+        relatedPullRequest: pullRequest._id
+      };
+
+      _.forEach(committedReviewers, function(reviewer) {
+        shall.grant(reviewer.username, achievement);
+      });
+    }
+  }
+};
+
+
+function reviewersWhoCommittedToPullRequest(pullRequest) {
+  var committedReviewers = [];
+
+  _.forEach(pullRequest.commits, function(commit) {
+    if (_.includes(pullRequest.reviewers, commit.creator.username)) {
```

Comment:
Sure. added to `PullRequestExample.MD`. also, if you test things locally, everything is written really nicely (that's where I copy things to `PullReuqestExample.MD`) at `<local_url>/logs`

The `creator` you see everywhere is the `pullRequest.creator`. for commits, there's a `commit.author`, which is the person who wrote the commit, and a `commit. committer`, which is the person who pushed the commit.